tools/devops/automation/templates/build/build.yml

```
@@ -393,6 +393,9 @@ steps:
 
       echo "Rollback file contents:" 
       echo "$(cat $WORKLOAD_DST)"
+      
+      echo "##vso[task.setvariable variable=WorkloadVersion;]$IOS_NUGET_VERSION_FULL"
```

The versions are different between all platforms, you need to set a variable for each of them.

---

I was hoping for the opposite. Probably I should do $(PLATFORM)_WORLOAD_VERSION. And change the loop or env def in the makefile

---

Part 2:

```suggestion
      echo "##vso[task.setvariable variable=iOSWorkloadVersion;]$IOS_NUGET_VERSION_FULL"
```

needs to be expanded for all platforms.